ios-swiftui/README.md

```
@@ -0,0 +1,169 @@
+| `sherpa-onnx-shared` | Shared library (`.dylib` in xcframework) | `import SherpaOnnxShared` |
+
+## Using SPM (Swift Package Manager)
+
+There are two ways to add sherpa-onnx to your Xcode project:
+
+### Option A: Remote Package (from GitHub)
+
+1. In Xcode, open your project
+2. Go to **File → Add Package Dependencies...**
+3. Enter the repository URL:
+   ```
```

markdownlint-cli2 (0.23.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)